elements/lisk-chain/test/unit/event.spec.ts

```diff
@@ -104,4 +104,44 @@ describe('event', () => {
 			expect(event.toObject()).toEqual(eventObj);
 		});
 	});
+
```

Comment:
Maybe we can still add a failing test? Like if `eventIndex >= 2**EVENT_TOPIC_INDEX_LENGTH_BITS`

Reply:
if event index is greater than `2 ** 30` then `((this._index << EVENT_TOPIC_INDEX_LENGTH_BITS) >>> 0)` becomes `0` and it gives the wrong value. 
But the index is checked with `MAX_EVENTS_PER_BLOCK`, so this will never happen.

Reply:
Sounds good.